# HG changeset patch
# User Jorg Knobloch
# Parent  f64069b6b999b161f5a5a352206339712ef9a3bf
Branding: Set UA string to Thunderbird, is derived from MOZ_APP_BASENAME.

diff --git a/mail/moz.configure b/mail/moz.configure
--- a/mail/moz.configure
+++ b/mail/moz.configure
@@ -26,8 +26,9 @@ else:
 # for application.ini's "Name" field, which controls profile location in
 # the absence of a "Profile" field, and various system
 # integration hooks (Unix remoting, Windows MessageWindow name, etc.)
 imply_option("MOZ_APP_BASENAME", "Betterbird")
+imply_option("MOZ_APP_UA_NAME", "Thunderbird")
 set_config("MOZ_APPUPDATE_HOST", "aus.thunderbird.net")
 
 imply_option("--enable-default-browser-agent", False)
 imply_option("MOZ_REQUIRE_SIGNING", False)
